lib/trento_web/plugs/app_jwt_auth_plug.ex

```diff
@@ -42,7 +51,13 @@ defmodule TrentoWeb.Plugs.AppJWTAuthPlug do
     The generated credentials will be stored in private section of the Plug.Conn struct
   """
   def create(conn, user, _config) do
-    claims = %{"sub" => user.id}
+    {:ok, user} = Users.get_user(user.id)
+
+    claims = %{
```

Comment:
Please use this claims only for access token, for refresh token on line 62 just use the sub as claims `%{"sub" => user.id}`, as you can see on line 83 are the only one we need when dealing with a refresh token, no further user infos are needed for this type of tokens

Reply:
Thanks for pointing out! Here we go https://github.com/trento-project/web/pull/3240/commits/b0c77b56b509ed4c5599c7ec6888ce5e3a89cbf2